Model/Configuration.php

```diff
@@ -63,6 +63,11 @@ public function __construct(
         }
     }
 
+    public function getStoreId()
+    {
+        return $this->storeId;
+    }
```

Comment:
Will this work? Is `storeId` still set on the Configuration?

Reply:
I don't know!

The PR has been updated to remove the setter from the config.